File: src/tools/RandomWord.py
from random_word import RandomWords

r = RandomWords()
def random_sentence():
    a = r.get_random_word(includePartOfSpeech="adjective", excludePartOfSpeech="noun")
    while (not a) :
        a = r.get_random_word(includePartOfSpeech="adjective", excludePartOfSpeech="noun")
    b = r.get_random_word(excludePartOfSpeech="adjective,adverb,verb")
    while (not b) :
        b = r.get_random_word(excludePartOfSpeech="adjective,adverb,verb")
    c = r.get_random_word(excludePartOfSpeech="adjective,noun,verb")
    while (not c) :
        c = r.get_random_word(excludePartOfSpeech="adjective,noun,verb")
    d = r.get_random_word(excludePartOfSpeech="adjective,noun,adverb")
    while (not d) :
        d = r.get_random_word(excludePartOfSpeech="adjective,noun,adverb")
    e = r.get_random_word(includePartOfSpeech="adjective")
    while (not e) :
        e = r.get_random_word(includePartOfSpeech="adjective")
    f = r.get_random_word(excludePartOfSpeech="adjective,adverb,verb")
    while (not f) :
        f = r.get_random_word(excludePartOfSpeech="adjective,adverb,verb")
    print("The", a, b, c, d,"the", e, f)

random_sentence()

File: src/tools/RandomWord.js
const puppeteer = require('puppeteer');

{/* <>
<div src = "" id = "adjectiveOne" />
<div src = "" id = "nounOne" />
<div src = "" id = "adverb" />
<div src = "" id = "verb" />
<div src = "" id = "adjectiveTwo" />
<div src = "" id = "nounTwo" />
</> */}
 
async function adjective(url) {
    const browser = await puppeteer.launch();
    const page = await browser.newPage();
    await page.goto(url);

    const [el] = await page.$x('//*[@id="result"]/li/div/span');
    const src = await el.getProperty('src');

    console.log({el});

    browser.close();
}
adjective('https://randomwordgenerator.com/adjective.php');

// console.log("about to fectch first noun");
// console.log("about to fetch adverb");
// console.log("about to fetch verb");
// console.log("about to fetch second adjective");
// console.log("about to fetch second noun");
